examples/examples.test.ts

```diff
@@ -1,13 +1,20 @@
 import {assert} from 'chai';
 import * as vl from '../src/vl';
 
-const ZSchema = require('z-schema');
+const zSchema = require('z-schema');
 const inspect = require('util').inspect;
 const dl = require('datalib');
 const fs = require('fs');
 const path = require('path');
 
-const validator = new ZSchema({
+zSchema.registerFormat('color', function (str) {
+  return false;
```

Comment:
The difference is that from where you copied it, we only care that the validator exists. Here the format validator should actually check whether `red` is a color. 

I guess we never specify a color in an example?